website/content/en/docs/deployment/cognito/guide.md

```diff
@@ -104,40 +104,44 @@ From this point onwards, we will be creating/updating the DNS records **only in
     1. **[Option 2]** Install individual components:
         ```bash
         # Kubeflow namespace
-        kustomize build common/kubeflow-namespace/base | kubectl apply -f -
+        kustomize build upstream/common/kubeflow-namespace/base | kubectl apply -f -
         
         # Kubeflow Roles
-        kustomize build common/kubeflow-roles/base | kubectl apply -f -
+        kustomize build upstream/common/kubeflow-roles/base | kubectl apply -f -
         
         # Istio
-        kustomize build common/istio-1-9/istio-crds/base | kubectl apply -f -
-        kustomize build common/istio-1-9/istio-namespace/base | kubectl apply -f -
-        kustomize build common/istio-1-9/istio-install/base | kubectl apply -f -
+        kustomize build upstream/common/istio-1-9/istio-crds/base | kubectl apply -f -
+        kustomize build upstream/common/istio-1-9/istio-namespace/base | kubectl apply -f -
+        kustomize build upstream/common/istio-1-9/istio-install/base | kubectl apply -f -
         
         # Cert-Manager
-        kustomize build common/cert-manager/cert-manager/base | kubectl apply -f -
-        kustomize build common/cert-manager/kubeflow-issuer/base | kubectl apply -f -
+        kustomize build upstream/common/cert-manager/cert-manager/base | kubectl apply -f -
+        kustomize build upstream/common/cert-manager/kubeflow-issuer/base | kubectl apply -f -
         
         # KNative
-        kustomize build common/knative/knative-serving/overlays/gateways | kubectl apply -f -
-        kustomize build common/knative/knative-eventing/base | kubectl apply -f -
-        kustomize build common/istio-1-9/cluster-local-gateway/base | kubectl apply -f -
+        kustomize build upstream/common/knative/knative-serving/overlays/gateways | kubectl apply -f -
+        kustomize build upstream/common/knative/knative-eventing/base | kubectl apply -f -
+        kustomize build upstream/common/istio-1-9/cluster-local-gateway/base | kubectl apply -f -
         
         # Kubeflow Istio Resources
-        kustomize build common/istio-1-9/kubeflow-istio-resources/base | kubectl apply -f -
+        kustomize build upstream/common/istio-1-9/kubeflow-istio-resources/base | kubectl apply -f -
         
         # Kubeflow Pipelines
         # reapply manifest if you see an error
-        kustomize build apps/pipeline/upstream/env/cert-manager/platform-agnostic-multi-user | kubectl apply -f -
-        
-        # KFServing
-        kustomize build apps/kfserving/upstream/overlays/kubeflow | kubectl apply -f -
+        kustomize build upstream/apps/pipeline/upstream/env/cert-manager/platform-agnostic-multi-user | kubectl apply -f -
         
+        # Kserve
+        kustomize build upstream/contrib/kserve/kserve | kubectl apply -f -
```

Comment:
why is this different from `awsconfigs/apps/kserve` in the deployments folder manifests?